app/javascript/mastodon/reducers/mutes.js

```diff
@@ -21,6 +23,8 @@ export default function mutes(state = initialState, action) {
     });
   case MUTES_TOGGLE_HIDE_NOTIFICATIONS:
     return state.updateIn(['new', 'notifications'], (old) => !old);
+  case MUTES_CHANGE_DURATION:
+    return state.updateIn(['new', 'duration'], (old) => Number(action.duration));
```

Comment:
`return state.setIn(['new', 'duration'], Number(action.duration);`

Reply:
I fix the issue.